src/plugins/data_source_management/public/components/utils.test.ts

```diff
@@ -380,5 +380,71 @@ describe('DataSourceManagement: Utils.ts', () => {
 
       expect(deepEqual(registedAuthTypeCredentials, expectExtractedAuthCredentials));
     });
+
+    test('Should inherit value from registered field when credentail state not have registered field', () => {
```

Comment:
Typo: `credentail`. Actually, there are only 8 files referenced by `crendentialFormField`. Can you please fix `crendentialFormField` in this PR, and then fix https://github.com/opensearch-project/OpenSearch-Dashboards/pull/6122#discussion_r1523982210 (31 files referenced)?

Reply:
Sure, thanks for the comment. 
Fixing.

Reply:
Typo resolved, thanks.